source/get-started/bare-metal-install-desktop.rst

```diff
@@ -61,28 +62,44 @@ these steps.
 
 .. _preliminary-steps-install-desktop-end:
 
-#. Select :guilabel:`Clear Linux OS` in the boot menu, shown in Figure 1.
+Choose boot menu option
+=======================
 
-   .. figure:: /_figures/bare-metal-install-desktop/bare-metal-install-desktop-01.png
-      :scale: 100%
-      :alt: Clear Linux OS in boot menu
+#. Choose one of the options shown in Figure 1.
+
+   a. Follow `Verify integrity of installer media (optional)`_.
+
+   #. Select :guilabel:`Clear Linux OS` in the boot menu.
+
+      .. figure:: /_figures/bare-metal-install-desktop/bare-metal-install-desktop-01.png
+         :scale: 100%
+         :alt: Clear Linux OS in boot menu
 
-      Figure 1: Clear Linux OS in boot menu
+         Figure 1: Clear Linux OS in boot menu
+
+      .. note::
+
+         If no action is taken, installation starts by default.
```

Comment:
Technically, the installation will not start, it just boots into the live image.

```suggestion
         If no action is taken, the live image starts by default.
```

Reply:
@mdhorn Thanks for the attention to wording. I applied your suggestions. See latest commit.